vm/jitdump.h

```diff
@@ -0,0 +1,66 @@
+// Below structs cribbed from tools/perf/util/jitdump.h in the Linux source tree.
```

Comment:
Do we need any licensing stuff in this file?

Reply:
The header describes itself as GPLv2-only, so wouldn't be compliant/correct if we were to assert that here. The general consensus ([FSF](http://lkml.iu.edu/hypermail/linux/kernel/0301.1/0362.html), current state of Oracle v Google) is that pure data structs like this aren't copyrightable though, so I think just attribution like this is the right option.

Reply:
ACK